core/server/master/src/main/java/alluxio/master/metrics/DefaultMetricsMaster.java

```diff
@@ -146,10 +146,15 @@ private void registerAggregators() {
         MetricsSystem.InstanceType.CLIENT, ClientMetrics.BYTES_READ_LOCAL_THROUGHPUT));
 
     // multi-value aggregators
-    addAggregator(new SingleTagValueAggregator(MetricsSystem.InstanceType.WORKER,
-        WorkerMetrics.BYTES_READ_UFS, WorkerMetrics.TAG_UFS));
-    addAggregator(new SingleTagValueAggregator(MetricsSystem.InstanceType.WORKER,
-        WorkerMetrics.BYTES_WRITTEN_UFS, WorkerMetrics.TAG_UFS));
+    addAggregator(new SingleTagValueAggregator(WorkerMetrics.BYTES_READ_UFS,
+        MetricsSystem.InstanceType.WORKER, WorkerMetrics.BYTES_READ_UFS, WorkerMetrics.TAG_UFS));
+    addAggregator(new SingleTagValueAggregator(WorkerMetrics.BYTES_WRITTEN_UFS,
+        MetricsSystem.InstanceType.WORKER, WorkerMetrics.BYTES_WRITTEN_UFS, WorkerMetrics.TAG_UFS));
+    for (WorkerMetrics.UfsOps ufsOp : WorkerMetrics.UfsOps.values()) {
+      addAggregator(new SingleTagValueAggregator(WorkerMetrics.UFS_OP_PREFIX + ufsOp,
+          MetricsSystem.InstanceType.MASTER, ufsOp.toString(),
+          WorkerMetrics.TAG_UFS));
```

Comment:
This aggregator is different from the throughput ones which are separated per mount point. For example, if there are two s3 mount, we still only keep track of one s3 metric.

Reply:
done